freemocap/tests/test_geometry_utilities.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import pytest
+
+from freemocap.utilities.geometry.project_3d_data_to_z_plane import project_3d_data_to_z_plane
+from freemocap.utilities.geometry.rotate_by_90_degrees_around_x_axis import rotate_by_90_degrees_around_x_axis
+
+
+def test_rotate_by_90_degrees_around_x_axis():
```

Comment:
these tests look good, but could you run through the machine along with a "make the assert statements print useful things and add logging or whatever" kind of prompt?